zebrad/src/commands/start/sync.rs

```diff
@@ -59,7 +59,7 @@ where
     ///  - state: the zebra-state that stores the chain
     ///  - verifier: the zebra-consensus verifier that checks the chain
     pub fn new(chain: Network, peers: ZN, state: ZS, verifier: ZV) -> Self {
-        let retry_peers = Retry::new(RetryLimit::new(3), peers.clone());
+        let retry_peers = Retry::new(RetryLimit::new(10), peers.clone());
```

Comment:
This is fine to merge but we might want to revisit it later -- the retry limit is a tradeoff between reducing errors from peers who don't give us data we should get (on the one hand) and accepting errors from peers who don't give us data we shouldn't get, if someone feeds us a bunch of bad hashes (on the other hand).  This doesn't actually work but it's because our sync logic is broken.